CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
-* Rename 'grade-a' and 'grade-c' to 'support-modern' and 'support-basic'
+## 0.11.0 / 2021-06-03
+* Add "declaration-property-unit-disallowed-list" with `px` for `font-size` and `line-height` (Volker E)
+* build: Update dependencies/devDependencies to latest (Ed Sanders)
+* build: Rename 'grade-a' and 'grade-c' to 'support-modern' and 'support-basic' (Volker E)
```

Comment:
This isn't a mere build artefact, it's a major breaking change.

Reply:
Oh, we should have aliases for the old ones

Reply:
See #141